docs/fundamentals/code-analysis/includes/ignore-ivt.md

```diff
@@ -6,4 +6,7 @@ By default, this rule is disabled if the assembly being analyzed uses <xref:Syst
 dotnet_code_quality.CAXXXX.ignore_internalsvisibleto = true
 ```
 
+> [!NOTE]
+> The `XXXX` part in the `CAXXXX` should be replaced with the number of rule it corresponds to.
```

Comment:
```suggestion
> Replace the `XXXX` part of `CAXXXX` with the ID of the applicable rule.
```

Reply:
Sure and Thanks for taking a look!

Reply:
Not that it's any of my business, but you could also just have changes the snippet to say `CA{rule-id}` instead of `CAXXXX`.
I think that would have been just as clear.
But an explanation is fine as well.